machine.go

```diff
@@ -1085,10 +1089,26 @@ func (m *Machine) CreateSnapshot(ctx context.Context, memFilePath, snapshotPath
 	return nil
 }
 
+// LoadSnapshot load a snapshot
+func (m *Machine) LoadSnapshot(ctx context.Context, memFilePath, snapshotPath string, opts ...LoadSnapshotOpt) error {
```

Comment:
It would be great if we don't have to expose the method. LoadSnapshot() would fail most of the case once the VM is started.

That being said, I don't have a good idea right now.

Reply:
That's correct, it's not very useful the way it is now.